src/main/java/net/dzikoysk/funnyguilds/basic/rank/RankUtils.java

```diff
@@ -112,7 +112,7 @@ public static int getIndex(String var) {
         try {
             result = Integer.parseInt(sb.toString());
         } catch(NumberFormatException e) {
-            FunnyGuilds.getInstance().getPluginLogger().parser(var + " contains an invalid number: " + sb.toString());
+            FunnyGuilds.getPluginLogger().parser(var + " contains an invalid number: " + sb.toString());
```

Comment:
I'd suggest to rename `var` variable since it's keyword in newer java versions and surely will blow up when someone tries to compile against latest version